.gitpod.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gitpod/workspace-python
+
+RUN pyenv install 3.9.13 \
+    && pyenv global 3.9.13
+
+# # Python Dependencies
+# ARG PYTHON_VERSION=3.9.13
+
+# RUN sudo add-apt-repository ppa:deadsnakes/ppa -y
+# RUN sudo apt install python${PYTHON_VERSION} -y
+# RUN sudo apt install python${PYTHON_VERSION}-venv -y
+# RUN sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VERSION} 2
+# RUN sudo pip install --upgrade pip
```

Comment:
Why did these have to be commented out?

Reply:
they were causing the gitpod load to fail. This change make the gitpod load quicker and still accomplishes the python version load that we want. I left the commented out code there for reference in case we still wanted any of it. Is there something there you want to revive?

Reply:
Curious about why this was causing issues for you but not with the aac-spec repo. As long as you're tracking it as an issue and this is currently working, we can evaluate it as we continue to work in the repository and maintain what a rollback would look like with the commented section. This should be okay for now.